idp-service/src/test/java/io/mosip/idp/flows/AuthorizationAPIFlowTest.java

```diff
@@ -126,13 +126,13 @@ public void init() throws Exception {
     }
 
     @Test
-    public void authorizationCodeFlowTest_Invalid_clientId() throws Exception {
+    public void authorizationCodeFlowTest_invalidClientId_thenFail() throws Exception {
```

Comment:
since the class file name is authorizationCodeFlowTest, do we need that in every unit test method name ?

Reply:
removed.